client/src/app/site/motions/modules/motion-block/components/motion-block-list/motion-block-list.component.ts

```diff
@@ -186,6 +200,6 @@ export class MotionBlockListComponent extends BaseListViewComponent<ViewMotionBl
      * Cancels the current form action
      */
     public onCancel(): void {
```

Comment:
Not required anymore. Use the dialogs information do detect `onClose`-events without information